meilisearch-http/tests/search_settings.rs

```diff
@@ -228,98 +176,72 @@ async fn search_with_settings_synonyms() {
         "proximity",
         "attribute",
         "wordsPosition",
-        "desc(popularity)",
+        "desc(age)",
         "exactness",
-        "desc(vote_average)"
+        "desc(balance)"
       ],
       "distinctAttribute": null,
       "searchableAttributes": [
-        "title",
-        "tagline",
-        "overview",
-        "cast",
-        "director",
-        "producer",
-        "production_companies",
-        "genres"
+        "name",
+        "age",
+        "color",
+        "gender",
+        "email",
+        "address",
+        "about"
       ],
       "displayedAttributes": [
-        "title",
-        "director",
-        "producer",
-        "tagline",
-        "genres",
-        "id",
-        "overview",
-        "vote_count",
-        "vote_average",
-        "poster_path",
-        "popularity"
+        "name",
+        "age",
+        "gender",
+        "color",
+        "email",
+        "phone",
+        "address",
+        "balance"
       ],
       "stopWords": null,
       "synonyms": {
-        "avangers": [
-          "Captain America",
-          "Iron Man"
-        ]
+          "exarcitation": [
+              "exercitation"
+          ]
```

Comment:
uhm, I think this synonym is too close to the original word, and actually is caught by the typo system. Can you try with a word that is unrelated to make sure the correct documents are picked?

Reply:
Oh, my bad. I can understand now. Will update it with a different word. 👍